src/plugins/media/schema.js

```diff
@@ -42,10 +42,16 @@ function normalizeMedia(change, error) {
 export default function schema({ inlines }) {
     return {
         blocks: {
-            // TODO: do we actually want to keep multiple versions?
```

Comment:
This is still an unresolved question.

Reply:
This is about `for`, right? It seems like using this attribute is right way to do this